### ISSUE TITLE
demo api issue

### ISSUE THREAD

**Issue by glaszig** [cachethq/Cachet#913]
what am i doing wrong?

```
$ /usr/bin/curl -H 'Accept: application/json' -H 'Content-Type: application/json' \
-H 'X-Cachet-Token: 9yMHsdioQosnyVK4iCVR' \
-d '{ "name": "Unreachable", "status" : 1, "component_id" : 1, "notify" : true }' \
https://demo.cachethq.io/api/v1/incidents

{"errors":[{"id":"30aa4d70-2953-4447-b640-3d5e52701c60","status":400,"title":"Bad Request","detail":"The request cannot be fulfilled due to bad syntax."}]}r
```

**Comment by jbrooksuk** [comment]
I think that `notify` may need to be `11.

**Comment by glaszig** [comment]
did not help. that'd be weird anyway. `true`, `false` are native json types.

**Comment by jbrooksuk** [comment]
Ah, you have enabled subscribers correctly?

**Comment by glaszig** [comment]
> Ah, you have enabled subscribers correctly?

i'm testing this line against your demo server. on there, there's only one unverified subscriber. but subscribers aren't my issue, are they? i mean, the error message doesn't say that.

what i get on my server is this (cachet log):

```
[2015-08-16 17:31:03] local.WARNING: exception 'Symfony\Component\HttpKernel\Exception\BadRequestHttpException' in /var/www/html/app/Http/Controllers/Api/IncidentController.php:89
Stack trace:
#0 [internal function]: CachetHQ\Cachet\Http\Controllers\Api\IncidentController->postIncidents(Object(Illuminate\Auth\Guard))
#1 /var/www/html/bootstrap/cache/compiled.php(8492): call_user_func_array(Array, Array)
#2 /var/www/html/bootstrap/cache/compiled.php(8561): Illuminate\Routing\Controller->callAction('postIncidents', Array)
#3 /var/www/html/bootstrap/cache/compiled.php(8541): Illuminate\Routing\ControllerDispatcher->call(Object(CachetHQ\Cachet\Http\Controllers\Api\IncidentController), Object(Illuminate\Routing\Route), 'postIncidents')
#4 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#5 /var/www/html/bootstrap/cache/compiled.php(9198): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#6 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#7 /var/www/html/bootstrap/cache/compiled.php(9180): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#8 /var/www/html/bootstrap/cache/compiled.php(8542): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#9 /var/www/html/bootstrap/cache/compiled.php(8527): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(CachetHQ\Cachet\Http\Controllers\Api\IncidentController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'postIncidents')
#10 /var/www/html/bootstrap/cache/compiled.php(7502): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'CachetHQ\\Cachet...', 'postIncidents')
#11 /var/www/html/bootstrap/cache/compiled.php(7473): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#12 /var/www/html/bootstrap/cache/compiled.php(7126): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#13 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#14 /var/www/html/bootstrap/cache/compiled.php(9198): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#15 /var/www/html/app/Http/Middleware/ApiAuthenticate.php(65): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#16 [internal function]: CachetHQ\Cachet\Http\Middleware\ApiAuthenticate->handle(Object(Illuminate\Http\Request), Object(Closure))
#17 /var/www/html/bootstrap/cache/compiled.php(9190): call_user_func_array(Array, Array)
#18 /var/www/html/app/Http/Middleware/Acceptable.php(34): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#19 [internal function]: CachetHQ\Cachet\Http\Middleware\Acceptable->handle(Object(Illuminate\Http\Request), Object(Closure), 'application/jso...')
#20 /var/www/html/bootstrap/cache/compiled.php(9190): call_user_func_array(Array, Array)
#21 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#22 /var/www/html/bootstrap/cache/compiled.php(9180): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#23 /var/www/html/bootstrap/cache/compiled.php(7127): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#24 /var/www/html/bootstrap/cache/compiled.php(7115): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#25 /var/www/html/bootstrap/cache/compiled.php(7100): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#26 /var/www/html/bootstrap/cache/compiled.php(2089): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#27 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#28 /var/www/html/bootstrap/cache/compiled.php(9198): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#29 /var/www/html/bootstrap/cache/compiled.php(12450): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 /var/www/html/bootstrap/cache/compiled.php(9190): call_user_func_array(Array, Array)
#32 /var/www/html/bootstrap/cache/compiled.php(11099): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#34 /var/www/html/bootstrap/cache/compiled.php(9190): call_user_func_array(Array, Array)
#35 /var/www/html/bootstrap/cache/compiled.php(12188): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#36 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#37 /var/www/html/bootstrap/cache/compiled.php(9190): call_user_func_array(Array, Array)
#38 /var/www/html/bootstrap/cache/compiled.php(12127): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#39 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#40 /var/www/html/bootstrap/cache/compiled.php(9190): call_user_func_array(Array, Array)
#41 /var/www/html/bootstrap/cache/compiled.php(2760): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#43 /var/www/html/bootstrap/cache/compiled.php(9190): call_user_func_array(Array, Array)
#44 /var/www/html/vendor/fideloper/proxy/src/TrustProxies.php(47): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#45 [internal function]: Fideloper\Proxy\TrustProxies->handle(Object(Illuminate\Http\Request), Object(Closure))
#46 /var/www/html/bootstrap/cache/compiled.php(9190): call_user_func_array(Array, Array)
#47 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#48 /var/www/html/bootstrap/cache/compiled.php(9180): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#49 /var/www/html/bootstrap/cache/compiled.php(2036): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#50 /var/www/html/bootstrap/cache/compiled.php(2019): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#51 /var/www/html/public/index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#52 {main} {"identification":{"id":"57df4de6-dc3d-4f83-b502-0487d2a0ba60"}}
```

**Comment by glaszig** [comment]
oh. i see. the docs are wrong :smile_cat: 

[docs](https://docs.cachethq.io/docs/incidents) vs [code](https://github.com/cachethq/Cachet/blob/1.2/app/Models/Incident.php#L30): not only `name` and `status` are required but also `visible` and `message`.

**Comment by jbrooksuk** [comment]
Could you suggest an edit on the page please?

**Comment by glaszig** [comment]
i tried. but readme.io is throwing "error submitting form" without any explanation at me.

**Comment by jbrooksuk** [comment]
Closed by 9219223.
